helm/oauth2-proxy/README.md

```diff
@@ -60,6 +60,10 @@ See the [v1.22 API deprecations guide](https://kubernetes.io/docs/reference/usin
 
 For the same reason `service.port` was renamed to `service.portNumber`.
 
+### To 5.0.0
+
+Version 5.0.0 introduces support for custom labels and refactor [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). This is a breaking change.
```

Comment:
This message needs to be expanded to explain what the breaking change actually does, what is the effect and how to migrate from version 4 to 5

Reply:
I agree

Reply:
Version 5.0.0 introduces support for custom labels and refactor [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/). This is a breaking change because many labels of all resources need to be updated to stay consistent.


Is it ok as explanation ?

Reply:
an "how to upgrade" is needed. Maybe things like https://github.com/bitnami/charts/tree/master/bitnami/redis#to-400 will work, maybe it is needed to uninstall.

Reply:
@nlamirault the readme file should be integrated with a "How to upgrade" section. It is the last step to be able to merge the PR. Could you please do that?

P.S.: I solved the conflict for you.